lib/src/common/variables.ts

```diff
@@ -717,340 +549,340 @@ export const componentTokens = {
+    fontColorOnDark: CORE_TOKENS.color_white,
     display: "block",
-    fontSize: globalTokens.type_scale_03,
-    fontWeight: globalTokens.type_regular,
+    fontSize: CORE_TOKENS.type_scale_03,
+    fontWeight: CORE_TOKENS.type_regular,
   },
   progressBar: {
-    trackLineColor: globalTokens.hal_purple_s_38,
-    trackLineColorOnDark: globalTokens.hal_purple_l_65,
-    totalLineColor: globalTokens.hal_grey_l_90,
-    labelFontFamily: globalTokens.type_sans,
-    labelFontSize: globalTokens.type_scale_01,
-    labelFontStyle: globalTokens.type_normal,
-    labelFontWeight: globalTokens.type_regular,
-    labelFontColor: globalTokens.hal_black,
-    labelFontColorOnDark: globalTokens.hal_white,
-    labelFontTextTransform: globalTokens.type_uppercase,
-    valueFontFamily: globalTokens.type_sans,
-    valueFontSize: globalTokens.type_scale_01,
-    valueFontStyle: globalTokens.type_normal,
-    valueFontWeight: globalTokens.type_regular,
-    valueFontColor: globalTokens.hal_black,
-    valueFontColorOnDark: globalTokens.hal_white,
-    valueFontTextTransform: globalTokens.type_uppercase,
-    helperTextFontColor: globalTokens.hal_black,
-    helperTextFontColorOnDark: globalTokens.hal_white,
-    helperTextFontSize: globalTokens.type_scale_01,
-    helperTextFontStyle: globalTokens.type_normal,
-    helperTextFontWeight: globalTokens.type_regular,
-    helperTextFontFamily: globalTokens.type_sans,
+    trackLineColor: CORE_TOKENS.color_purple_700,
+    trackLineColorOnDark: CORE_TOKENS.color_purple_500,
+    totalLineColor: CORE_TOKENS.color_grey_200,
+    labelFontFamily: CORE_TOKENS.type_sans,
+    labelFontSize: CORE_TOKENS.type_scale_01,
+    labelFontStyle: CORE_TOKENS.type_normal,
+    labelFontWeight: CORE_TOKENS.type_regular,
+    labelFontColor: CORE_TOKENS.color_black,
+    labelFontColorOnDark: CORE_TOKENS.color_white,
+    labelFontTextTransform: CORE_TOKENS.type_uppercase,
+    valueFontFamily: CORE_TOKENS.type_sans,
+    valueFontSize: CORE_TOKENS.type_scale_01,
+    valueFontStyle: CORE_TOKENS.type_normal,
+    valueFontWeight: CORE_TOKENS.type_regular,
+    valueFontColor: CORE_TOKENS.color_black,
+    valueFontColorOnDark: CORE_TOKENS.color_white,
+    valueFontTextTransform: CORE_TOKENS.type_uppercase,
+    helperTextFontColor: CORE_TOKENS.color_black,
+    helperTextFontColorOnDark: CORE_TOKENS.color_white,
+    helperTextFontSize: CORE_TOKENS.type_scale_01,
+    helperTextFontStyle: CORE_TOKENS.type_normal,
+    helperTextFontWeight: CORE_TOKENS.type_regular,
+    helperTextFontFamily: CORE_TOKENS.type_sans,
     thickness: "9px",
     borderRadius: "5px",
-    overlayColor: globalTokens.color_grey_a_800,
-    overlayFontColor: globalTokens.hal_white,
+    overlayColor: CORE_TOKENS.color_grey_800_a,
+    overlayFontColor: CORE_TOKENS.color_white,
   },
   quickNav: {
-    fontColor: globalTokens.hal_grey_s_40,
-    hoverFontColor: globalTokens.hal_purple_d_70,
-    dividerBorderColor: globalTokens.hal_grey_l_75,
```

Comment:
The previous value of the token did not exist in the Halstack colour palette. We change it to the closest valid colour.
![image](https://github.com/dxc-technology/halstack-react/assets/44321109/8a5972e9-1ae0-4272-8578-c31ac2e4dc79)

Reply:
This is a special case. `disabledBorderColor` doesn't apply any style to the regular component, it is a customizable token to add a border to the component when the state is disabled. So, by default, It doesn't matter its color since the border width is 0px and doesn't make any difference.